test/DaprExtensionTests/Tests/DaprBindingTests.cs

```diff
@@ -74,6 +74,41 @@ public async Task SendMessage_JsonElementAsyncCollector(object message)
             Assert.Equal(JsonSerializer.Serialize(expectedPayload, Utils.DefaultSerializerOptions), req.ContentAsString);
         }
 
+        [Theory]
+        [MemberData(nameof(GetObjectAsyncCollectorInputs))]
+        public async Task SendMessage_JsonElementValueKindStringAsyncCollector(object message)
+        {
+            string stringInput = $@"{{
+                        ""data"": {JsonSerializer.Serialize(message)},
+                        ""operation"": ""create"",
+                        ""metadata"": {{
+                            ""key"": ""myKey""
+                        }},
+                        ""bindingName"": ""myBinding""
+                   }}";
+
+            stringInput = stringInput.Replace("\r\n", string.Empty);
+            stringInput = stringInput.Replace("\n", string.Empty);
+            stringInput = stringInput.Replace("\"", "\\\"");
+            stringInput = "\"" + stringInput + "\"";
```

Comment:
Nit: comment here to explain why we are doing it, might not be obvious for new readers.

Reply:
Done!